spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/XsuaaServicesParser.java

```diff
@@ -93,14 +95,15 @@ public XsuaaServicesParser(String vcapServicesJson) {
 	 *
 	 */
 	public Properties parseCredentials() throws IOException {
-		Properties properties = new Properties();
-		JSONObject credentialsJsonObject = parseCredentials(vcapServices);
-		if (credentialsJsonObject != null) {
-			Set<String> keys = credentialsJsonObject.keySet();
-			for (String key : keys) {
-				properties.put(key, credentialsJsonObject.get(key).toString());
+		Properties properties;
+			properties = new Properties();
```

Comment:
pls apply formatter

Reply:
@nenaraab I'll do that on the branch where this will be merged into, otherwise there will be too many file changes.

Reply:
pls fix indentation